spec/unit/userActivity.spec.js

```diff
@@ -0,0 +1,202 @@
+const { UserActivityTracker, UserActivityTrackerConfig  } = require("../../lib/components/userActivity")
```

Comment:
```suggestion
const { UserActivityTracker, UserActivityTrackerConfig  } = require("../..");
```

This is personal preference, but I like to pull components from the package root because that tests that we are properly exporting the components (and if we move them, this file doesn't need to change).

Reply:
Good call!